ext/formats.d.ts

```diff
@@ -0,0 +1,10 @@
+import * as opentracing from 'opentracing'
+
+declare const formats: {
+  TEXT_MAP: typeof opentracing.FORMAT_TEXT_MAP
+  HTTP_HEADERS: typeof opentracing.FORMAT_HTTP_HEADERS
+  BINARY: typeof opentracing.FORMAT_BINARY
+  LOG: boolean
```

Comment:
This should be `log`.

Reply:
Don't know how I put that boolean there, but it's fixed.